### PR TITLE
DOC-850: Document new plugin commands

### PR DESCRIPTION
**Pull request by diogo405**
Related Ticket: 

Description of Changes:
* Added new commands - `mceEmoticons`, `mceTemplate`, and `mceWordCount` - to commands page 

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] ~`_data/nav.yml` has been updated (if applicable)~
- [x] Files has been included where required (if applicable)
_includes/commands/emoticons-cmds.md
_includes/commands/wordcount-cmds.md
- [x] ~Files removed have been deleted, not just excluded from the build (if applicable)~
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: release-notes/release-notes59.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ For information on the `toolbar_sticky_offset` option, see: [User interface opti
 
 {{site.productname}} 5.9 introduces the following minor new features:
 
-- changelog
+- Added new plugin commands: `mceEmoticons` (emoticons), `mceWordCount` (wordcount), and `mceTemplate` (template)
```
Comment (tylerkelly13) [inline release-notes/release-notes59.md:41]: Needs a bit more detail, like
```
- Added new open dialog commands for the following plugins:
  - Emoticons (`mceEmoticons`)
  - Word Count (`mceWordCount`)
  - Template (`mceTemplate`)
```

**Review by tylerkelly13 — CHANGES_REQUESTED** [review]
Release note needs work, otherwise LGTM!

File: _includes/commands/emoticons-cmds.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+| Command      | Description                        |
+| ------------ | ---------------------------------- |
+| mceEmoticons | Opens the Emoticons dialog. |
+
+**Examples**
```
Comment (lnewson) [inline _includes/commands/emoticons-cmds.md:6]:
```suggestion
**Example**
```

File: release-notes/release-notes59.md (hunk 1 above)
Comment (diogo405) [reply release-notes/release-notes59.md]: Right, changed 👍

File: _includes/commands/emoticons-cmds.md (hunk 2 above)
Comment (diogo405) [reply _includes/commands/emoticons-cmds.md]: Fixed.

**Review by tylerkelly13 — APPROVED** [review]

File: release-notes/release-notes59.md [hunk 3]
```diff
@@ -38,7 +38,10 @@ For information on the `toolbar_sticky_offset` option, see: [User interface opti
 
 {{site.productname}} 5.9 introduces the following minor new features:
 
-- changelog
+- Added new plugins commands:
+  - `mceEmoticons` opens Emoticons dialog
+  - `mceWordCount` displays the Word Count summary
+  - `mceTemplate` shows the Template selector dialog
```
Comment (lnewson) [inline release-notes/release-notes59.md:44]:
```suggestion
- Added new plugin commands:
  - `mceEmoticons` opens the Emoticons dialog.
  - `mceWordCount` displays the Word Count summary dialog.
  - `mceTemplate` shows the Template dialog.
```

File: _includes/commands/wordcount-cmds.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+
+| Command           | Description                                                                     |
+| ----------------- | ------------------------------------------------------------------------------- |
+| mceWordCount      | Opens the Word Count dialog. |
```
Comment (lnewson) [inline _includes/commands/wordcount-cmds.md:4]:
```suggestion
| mceWordCount      | Opens the Word Count summary dialog. |
```
This is to keep it consistent with what we've said in the release notes.

**Review by lnewson — APPROVED** [review]

**Review by EkimChau — APPROVED** [review]

**Pull request merged**